cmd/swarmctl/service/list.go

```diff
@@ -31,6 +31,17 @@ var (
 				return err
 			}
 
+			nr, err := c.ListNodes(common.Context(cmd), &api.ListNodesRequest{})
+			if err != nil {
+				return err
+			}
+			readyNodes := make(map[string]struct{})
+			for _, n := range nr.Nodes {
+				if n.Status.State == api.NodeStatus_READY {
```

Comment:
We generally force push with new changes. GitHub is relatively smart about hiding comments that apply to old code. They can still be displayed on request for context. It's not perfect, but it's preferable to opening lots of new PRs.